test/asammdf/gui/widgets/test_PlotWidget_Shortcuts.py

```diff
@@ -225,9 +223,253 @@ def test_Plot_Plot_Shortcut_Key_R(self):
         )
 
         # Press Key 'R' for range selection
-        QtTest.QTest.keyClick(plot.plot, QtCore.Qt.Key_R)
+        QtTest.QTest.keyClick(self.plot.plot, QtCore.Qt.Key_R)
         self.processEvents(timeout=0.01)
 
         # Save PixMap of clear plot
-        clear_pixmap = plot.plot.viewport().grab()
+        clear_pixmap = self.plot.plot.viewport().grab()
         self.assertTrue(Pixmap.is_black(clear_pixmap))
+
+    def test_Plot_Plot_Shortcut_Key_F(self):
+        """
+        Test Scope:
+            Check if 2 plotted signals is fitted on Y ax after pressing key "F".
+        Events:
+            - Open 'FileWidget' with valid measurement.
+            - Select 2 signals and create a plot
+            - Press Key "S" to separate signals (precondition)
+            - Press Key "F"
+        Evaluate:
+            - Evaluate that two signals are available
+            - Evaluate that plot is not black
+            - Evaluate that the color of signals is not displayed on top and bottom line of plot after pressing key "S"
+            - Evaluate that are both colors of signals in top and bottom line of plot after pressing key "F"
+        """
+        channel_0 = self.add_channel_to_plot(plot=self.plot, channel_index=7)
+        channel_1 = self.add_channel_to_plot(plot=self.plot, channel_name="ASAM.M.SCALAR.UBYTE.HYPERBOLIC")
```

Comment:
Do you still need these channels? It's possible to use the ones defined in `setUp` ?

Reply:
Nope, is one test without signals

Reply:
Key "R"